### ISSUE TITLE
top navigation bar

### ISSUE THREAD

**Issue by MTM828** [leahcimto/leahcimto.github.io#4]
this is something every site needs, why not add it?
u can see w3schools how to tutorial, but ill give you an idea and if u want, the code too.
you might need to learn more css, and i suggest creating a .css file which applies to all of ur sites pages, i remember u have to add <link rel="stylesheet" href="path/link to css file here"> in the... head i think.

anyways so it's basically a list of links, add display: inline; to the css so it's horizontal, blah blah add display:fixed; so its sticky blah blah

i understand that its like still empty and stuff so yeah, take ur time.
i can make the code for u if u wish.
and yeah.
idc about credits and wutever. and just saying, im new to github even tho i joined ages ago lol, i didnt have any motivation lol. so correct me if i do anything wrong :).

**Comment by leahcimto** [comment]
@MTM828 Thank you for the suggestion! I am pretty busy with school right now, but I will get around to work on this, probably on my winter break.

**Comment by MTM828** [comment]
Yeah I understand, and I might create a pull request as you suggested.

**Comment by leahcimto** [comment]
@MTM828 Fork this repository by pressing "Fork" located at the top right of the screen, make your edits on your fork, then press the compare button on your fork. There will be an option to make a pull request. Press it and create the PR. More info [here](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).
